docs/application/toc_all.md

```diff
@@ -305,7 +312,15 @@
 
 ## [Overview](/application/web/index.md)
 
-## Get Started
+## Get Started with Virtual Studio Code Extension
```

Comment:
"Virtual"

The  name of the Tools is Visual Studio, not Virtual Studio.
How about using the official name?

![image](https://user-images.githubusercontent.com/9958374/158321525-f52e6652-7189-475d-a4dc-9fb64d0ec0c9.png)